tests/ui/mock/alert_summary_very_big.json

```diff
@@ -0,0 +1,5194 @@
+{
+  "id": 30643,
+  "push_id": 936129,
+  "prev_push_id": 936120,
+  "created": "2021-07-23T11:12:07.794412",
+  "repository": "autoland",
+  "framework": 13,
+  "alerts": [
```

Comment:
If the summay is smaller there won't be a timeout on the js test (I've tried with 112 alerts and it passes)